### DISCUSSION TITLE
プライバシーポリシーの投稿について

### DISCUSSION THREAD

**Discussion by shiracamus** [increments/qiita-discussions#242] [要望・アイデア・不具合]
どうやら、スマホアプリのプライバシーポリシー設置場所としてQiitaが使われているようです。
Qiita本来の使い方から外れていると思うので、注意や対処（限定共有投稿にするなど）をしてはいかがでしょうか？

プライバシーポリシーの設置に関する知見を得た記事：
https://qiita.com/SNQ-2001/items/0ca6b2e56d8107f0fa1b

プライバシーポリシー記事の検索
キーワード： https://qiita.com/search?q=%E3%83%97%E3%83%A9%E3%82%A4%E3%83%90%E3%82%B7%E3%83%BC%E3%83%9D%E3%83%AA%E3%82%B7%E3%83%BC
タグ： https://qiita.com/tags/%e3%83%97%e3%83%a9%e3%82%a4%e3%83%90%e3%82%b7%e3%83%bc%e3%83%9d%e3%83%aa%e3%82%b7%e3%83%bc

**Comment by tanaka-asami415** [comment]
@shiracamus 
ご報告ありがとうございます🙏💦
利用規約違反になりますので対処方法を検討し、対応いたします🙇

**Reply by tanaka-asami415** [reply]
@shiracamus 
ご報告が遅くなり申し訳ありません。現時点で上記規約違反に該当する記事への対応が完了いたしました。
今後も引き続き対策を検討いたします🙇
